WordPress/src/main/java/org/wordpress/android/AppInitializer.kt

```diff
@@ -950,13 +950,8 @@ class AppInitializer @Inject constructor(
      */
     private inner class MemoryAndConfigChangeMonitor : ComponentCallbacks2 {
         override fun onConfigurationChanged(newConfig: Configuration) {
-            // If per-app locale is enabled make sure the in-app locale is correct,
-            // otherwise reapply in-app locale on configuration change
-            if (perAppLocaleManager.isPerAppLanguagePrefsEnabled()) {
-                perAppLocaleManager.checkAndUpdateOldLanguagePrefKey()
-            } else {
-                LocaleManager.setLocale(context)
```

Comment:
Question (❓): Seeing this `LocaleManager` related code being removed, I then searched for any other such occurrence in this class, finding only this `updateContextLocale(...)` function at the bottom, which only points to `WordPress.updateContextLocale(...)`, which only points to `JetpackMigrationLanguageUtil.applyLanguage(...)`, a utility class that was used during for the Jetpack migration period. My question is whether this is still relevant and could be also deleted as part of this overall feature (not necessarily part of this PR, just in general). 🤔

Reply:
Nice catch! Yes, this is being removed in a branch I'm already working on.

Reply:
Awesome, thanks Nick! 🚀